tests/test_files/sim_config.yaml

```diff
@@ -0,0 +1,39 @@
+  molecular_model:
```

Comment:
This is part of the "test meta data" that is uploaded. It makes sense to me to include it in the test to be uploaded.